src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -19,15 +19,15 @@
 #
 ################################
 
-# Base directory: C:\Code\sonarlint-visualstudio\src\Integration.Vsix\bin\Debug\VS2022\net472
+# Base directory: D:\repos\sq\slvs\src\Integration.Vsix\bin\Debug\VS2022\net472
```

Comment:
Originally this line was "D:\a\1\s\src\Integration.Vsix\bin\Release\VS2022\net472" shouldn't we change it back again? looks like it got messed up here https://github.com/SonarSource/sonarlint-visualstudio/commit/78e470a4430895fdc034ee0b56899953a65a8e64